frontend/src/modules/login/components/LoginForm.js

```diff
@@ -1,24 +1,48 @@
 import React from "react";
-import { Redirect } from 'react-router-dom';
-import { TextField, Button } from "@material-ui/core";
+import {Link, Redirect} from 'react-router-dom';
+import { TextField, Button, Box } from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
 import LoginFormBl from "./bl.js";
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+    margin: {
+        margin: theme.spacing(1),
+    },
+}));
+
 
 export default function LoginForm() {
+    const classes = useStyles();
     const { loggedIn, handleChange, handleSubmit, loginFailed } = LoginFormBl();
 
     return (
         <>
             {loggedIn ? <Redirect to="/" /> :
             <form onSubmit={handleSubmit}>
-                <h4>Log In</h4>
                 <div>
-                    <TextField type="text" name="username" label="Username" onChange={handleChange} />
+                    <TextField id = "filled-basic" label = "Filled" variant = "filled" type="text" name="username" label="Username" onChange={handleChange} />
                 </div>
                 <div>
-                    <TextField type="password" name="password" label="Password" onChange={handleChange} />
+                    <TextField id = "filled-basic" label = "Filled" variant = "filled" type="password" name="password" label="Password" onChange={handleChange} />
                 </div>
-                <Button type="submit">Submit</Button>
+                <Typography variant = "h3" className = {classes.margin}>
+                    No account? Sign up
+                    <Link to="/signup"> here.</Link>
+                </Typography>
+                <br></br>
+                <Button type="submit" variant = "contained" color = "primary" className = {classes.margin} style = {{maxWidth: '150px', minWidth: '150px'}}>Login</Button>
```

Comment:
I would break these tags and their parameters up across multiple lines for readability. There are multiple instances of this.